kimchi/src/circuits/constraints.rs

```diff
@@ -146,16 +146,16 @@ pub enum GateError {
     Custom { row: usize, err: String },
 }
 
-pub struct Builder<F: FftField> {
-    gates: Vec<CircuitGate<F>>,
-    sponge_params: ArithmeticSpongeParams<F>,
+pub struct Builder<G: KimchiCurve> {
```

Comment:
same here, do we need kimchicurve instead of FftField?

Reply:
Changed to F.